src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -230,163 +251,143 @@ const AnalysisCard = ({
   ])
 }
 
-const Analyses = _.flow(
-  forwardRefWithName('Analyses'),
-  requesterPaysWrapper({
-    onDismiss: () => Nav.history.goBack()
-  }),
-  wrapWorkspace({
-    breadcrumbs: props => breadcrumbs.commonPaths.workspaceDashboard(props),
-    title: 'Analyses',
-    activeTab: 'analyses'
-  }),
-  withViewToggle('analysesTab')
-)(({
-  name: workspaceName, namespace, workspace, workspace: { accessLevel, canShare, workspace: { cloudPlatform, workspaceId, googleProject, bucketName } },
+const activeFileTransferMessage: React.ReactElement<'div'> = div({
```

Comment:
For consistency, can `ReactElement` be imported individually instead of using the React namespace?

```ts
import { ReactElement } from 'react'

const el: ReactElement = ...
```

vs

```ts
import React from 'react'

const el: React.ReactElement = ...
```